### ISSUE TITLE
Troubleshoot guide addition

### ISSUE THREAD

**Issue by Lucantof** [starcitizen-lug/knowledge-base#16]
`/home/ubuntu/buildbot/runners/wine/wine-src/dlls/ntdll/unix/virtual.c :1571 : create_view:  l'assertion « !((UINT_PTR)base & page_mask) » a échoué.`

It's found in Lutris' wine logs of the game, and it indicates that vm.max_map_count hasn't been set high enough or at all.

**Comment by the-sane** [comment]
Good call. Do you think you can open a PR to add that error in? I’m super busy with work this week and that’d help me out a lot. We can start with the “create_view” part of the error (we don’t need the whole path part). I’ll approve it whenever I get a moment!

**Comment by Lucantof** [comment]
Alright, i'll try

**Comment by Lucantof** [comment]
Done, but i forgot to verify it. Hope it's fine anyway

**Comment by Lucantof** [comment]
Oh wait, i didn't notice you didn't want the full error but only the second part. Sorry i'll fix it

**Comment by the-sane** [comment]
Closed in #17; thanks!
